ungrib/src/rd_grib2.F

```diff
@@ -492,7 +492,7 @@ SUBROUTINE rd_grib2(junit, gribflnm, hdate,
               endif
 
               ! Scale lat/lon values to 0-180, default range is 1e6.
-              if (abs(map%lat1).ge.scale_factor) then
+              if (map%lat1.ge.scale_factor) then 
```

Comment:
@grafmi 
Michael,
Did we just change this and this is a revert?

Reply:
Yes, this seems to revert one of the changes made in PR #145 .

Reply:
Dave, sorry, I thought that I'm working with the newest version, but it seems that it was version 4.1. This has to be reverted back to `if (abs(map%lat1).ge.scale_factor) then` Thanks!

Reply:
I reverted the accidental change back to the current version. So this issue should be fixed.